test/unit/ngsiv2/examples/subscriptionRequests/bidirectionalSubscriptionRequest.json

```diff
@@ -0,0 +1,26 @@
+{
+  "subject":{
+    "entities": [
+      {
+        "id": "TheFirstLight",
+        "type": "TheLightType"
+      }
+    ],
+    "condition":{
+      "attrs":[
+        "location"
+      ]
+    }
+  },
+  "notification":{
+    "http":{
+      "url":"http://smartGondor.com/notify"
+    },
+    "attrs":[
+      "location"
+    ],
+    "attrsFormat": "normalized"
+  },
+  "expires": "2016-04-05T14:00:00.00Z",
```

Comment:
Looking to how bidirectional-plugin_test.js has ended at the end, I think it happens here as in simpleSubscriptionRequest.json. I mean, the field in calculated by the mock function and the "expired" field here should be removed.

Looking in particular to the following points:

https://github.com/dcalvoalonso/iotagent-node-lib/pull/4/files#diff-665dfc645ea8386f7fc737fc0476026fR107
https://github.com/dcalvoalonso/iotagent-node-lib/pull/4/files#diff-665dfc645ea8386f7fc737fc0476026fR157
https://github.com/dcalvoalonso/iotagent-node-lib/pull/4/files#diff-665dfc645ea8386f7fc737fc0476026fR217
https://github.com/dcalvoalonso/iotagent-node-lib/pull/4/files#diff-665dfc645ea8386f7fc737fc0476026fR340
https://github.com/dcalvoalonso/iotagent-node-lib/pull/4/files#diff-665dfc645ea8386f7fc737fc0476026fR340

As side comment, these 5 points could be also "decorated" with the comment used in subscription-tests.js, i.e.:

```
                    // Note that expired field is not included in the json used by this mock as it is a dynamic
                    // field. The following code performs such calculation and adds the field to the subscription
                    // payload of the mock.
```

Reply:
Fixed in cf32a5bf0a6de572d501273ce79a1a2fb76fd1cd